guides/common/modules/proc_configuring-installation.adoc

```diff
@@ -20,12 +20,14 @@ An internal label that matches the value is also created and cannot be changed a
 If you do not specify a value, an organization called *Default Organization* with the label *Default_Organization* is created.
 You can rename the organization name but not the label.
 
-* By default, all configuration files configured by the installer are managed by Puppet.
-When `{foreman-installer}` runs, it overwrites any manual changes to the Puppet managed files with the initial values.
+* By default, all configuration files configured by the installer are managed.
+When `{foreman-installer}` runs, it overwrites any manual changes managed files with the initial values.
```

Comment:
```suggestion
When `{foreman-installer}` runs, it overwrites any manual changes to the managed files with the initial values.
```

Reply:
That's what you get when you submit a PR in between other work.